test/e2e/tests/test_function.py

```diff
@@ -420,3 +420,59 @@ def test_function_package_type_image_with_signing_config(self, lambda_client):
 
         # Check Lambda function doesn't exist
         assert not lambda_validator.function_exists(resource_name)
+
+    def test_function_reconciler(self, lambda_client):
```

Comment:
Can you name this `test_function_is_synced` ?

Reply:
Sure

Reply:
Done